cpp/src/Slice/Grammar.h

```diff
@@ -36,16 +36,17 @@
    private implementation details that can be changed or removed.  */
 
 #ifndef YY_SLICE_SRC_SLICE_GRAMMAR_H_INCLUDED
-#define YY_SLICE_SRC_SLICE_GRAMMAR_H_INCLUDED
+# define YY_SLICE_SRC_SLICE_GRAMMAR_H_INCLUDED
```

Comment:
Is this code generated with a different version?

Reply:
No, but this is the first time these files have been touched since we `clang-format-ignored` them.
So, as of this PR, the files are exactly as `bison` generates them, without me reformatting them.